doc/languages.rst

```diff
@@ -1413,6 +1413,10 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
          The directory to use for the file library.
 
+      .. defcustom:: flycheck-ghdl-ieee-library
+
+         The standard to use for the ieee library
```

Comment:
Missing  a colon at sentence end.

Also, shouldn't `ieee` be `IEEE` when in plain text?  (Also in the docstring below).

Reply:
Fixed in latest commits.